lib/views/pr-detail-view.js

```diff
@@ -89,13 +80,13 @@ export class BareIssueishDetailView extends React.Component {
 
   constructor(props) {
     super(props);
-    autobind(this, 'handleRefreshClick', 'refresh', 'renderEmojiReactions', 'renderIssueBody', 'renderPullRequestBody');
+    autobind(this, 'handleRefreshClick', 'refresh', 'renderPullRequestBody', 'recordOpenInBrowserEvent');
```

Comment:
FYI I've been trying to convert `autobind` calls to the class property style that @kuychaco advocated in new code. So instead of:

```js
class SomeComponent extends React.Component {
  constructor(props) {
    super(props);
    autobind(this, 'boundMethod');
  }

  boundMethod() {
    console.log('You did the thing');
  }
}
```

It would be:

```js
class SomeComponent extends React.Component {
  boundMethod = () => {
    console.log('You did the thing');
  }
}
```

Turns out it apparently does the exact same thing, but requires one less import, can get rid of the need for a constructor entirely, and causes less duplication. (I still kind of miss our `@autobind` decoration, but I like having working code coverage even better). The only downside I've seen is that you can't use `cmd-R` to navigate to the definition.

(You don't _have_ to bother changing these here - this is just a general note for the future :smile:)